core/broadleaf-framework/src/main/java/org/broadleafcommerce/core/catalog/domain/ProductImpl.java

```diff
@@ -264,6 +264,13 @@ public class ProductImpl implements Product, ProductAdminPresentation, Status, A
             tooltip = "ProductImpl_Canonical_Url_Tooltip")
     protected String canonicalUrl;
 
+    @Column(name = "USE_DEFAULT_SKU_IN_INVENTORY")
```

Comment:
This needs to be setup as a conditional field.  @riteshadhikari17 Thoughts on this?  It makes things a bit more complex to make it conditional but we also don't want a schema change on a patch.

Reply:
Can we do conditional weave?

Reply:
We need to.  It does make the implementation a bit harder but it keeps up in line with the patch update process.  Do you have a good example that can be used?